pkg/runtime/wfengine/workflow.go

```diff
@@ -597,12 +597,21 @@ func (wf *workflowActor) runWorkflow(ctx context.Context, actorID string, remind
 			if errMarshal != nil {
 				return errMarshal
 			}
-
+			requestBytes := eventData
```

Comment:
@kaibocai This was a bug that got left out of #7308 and was causing errors in creating sub-orchestrations. Please take a look if this is the correct behavior while sending the policy to sub-orchestrations

Reply:
Creating a separate PR for this and removing it from here as it's P0 blocking other works.